### PR TITLE
Added scroll-to-top button

### PR DESCRIPTION
**Pull request by m-code12**
Fixes: #100 

Added a scroll-to-top button in homepage.

### CODE REVIEW

**Comment by m-code12** [comment]
@garg3133 Please review this pr.

File: templates/home/index.html [hunk 1]
```diff
@@ -112,6 +142,8 @@ <h1 data-aos="zoom-in" data-aos-duration="1500">Jagrati Promotes <span style="co
     </section>
     <!-- Showcase ends -->
 
+    <button onclick="topFunction()" id="myBtn" title="Go to top" class="ml-5"><i class="fa fa-angle-double-up" aria-hidden="true" style="color: black;font-weight: bolder;"></i></button>
```
Comment (garg3133) [inline templates/home/index.html:145]: Please put this button below the code for "Contact Us"  and above "Footer". Same thing for the CSS.

File: templates/home/index.html [hunk 2]
```diff
@@ -433,6 +465,23 @@ <h1 data-aos="fade-up">Contact Us</h1>
     <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/@glidejs/glide/dist/glide.js"></script>
     <script type="text/javascript" src='{% static "home/js/glide.js" %}'></script>
     <script src="https://unpkg.com/aos@2.3.1/dist/aos.js"></script>
+    <script>
+        
+        var mybutton = document.getElementById("myBtn");
```
Comment (garg3133) [inline templates/home/index.html:470]: Please use proper naming conventions. I think `scrollToTopButton` will do fine.

File: templates/home/index.html [hunk 3]
```diff
@@ -10,7 +10,37 @@
     <link href="https://unpkg.com/aos@2.3.1/dist/aos.css" rel="stylesheet">
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css" integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay" crossorigin="anonymous">
     <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/@glidejs/glide/dist/css/glide.core.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.4.0/css/font-awesome.min.css">
     <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/@glidejs/glide/dist/css/glide.theme.min.css">
+    <style type="text/css">
+    	#myBtn {
```
Comment (garg3133) [inline templates/home/index.html:16]: Please put this code in `style.css` file. Also, use the proper naming convention.

**Review by garg3133 — COMMENTED** [review]
I think why the CSS was not working after putting it in `style.css` file was because of the cached old CSS. Always clear the cache after making changes in CSS (reload the page using `CTRL + F5` to autoclear the cache while reloading).

**Comment by m-code12** [comment]
@garg3133 I have made all the changes that you suggested. Please review and merge this.

**Comment by garg3133** [comment]
Looks good @m-code12! Merging it into master.

Thank you for your contribution 🚀

**Pull request merged**